web/locales/en/messages.ftl

```diff
@@ -642,8 +642,8 @@ contribute-more =
        *[other] Ready to do { $count } more?
     }
 record-cta = Start recording
-record-platform-not-supported = We’re sorry, but your platform is not currently supported.
-record-platform-not-supported-desktop = On desktop computers, you can download the latest:
+record-platform-not-supported = We’re sorry, your platform is not currently supported.
+record-platform-not-supported-desktop = On desktop devices you can contribute by downloading...
```

Comment:
I didn't create new IDs for these because I felt like the changes were small enough that they don't merit retranslating in all the languages.